backend/src/main/java/capstone/backend/services/OrderToSupplierService.java

```diff
@@ -3,21 +3,29 @@
 import capstone.backend.exception.model.EntityNotFoundException;
 import capstone.backend.exception.model.EntityWithThisIdAlreadyExistException;
 import capstone.backend.mapper.OrderToSupplierMapper;
+import capstone.backend.model.dto.ProductDTO;
+import capstone.backend.model.dto.contact.SupplierDTO;
 import capstone.backend.model.dto.order.OrderToSupplierDTO;
 import capstone.backend.repo.OrderToSupplierRepo;
 import org.springframework.stereotype.Service;
+
 import java.util.List;
+
 import static capstone.backend.mapper.OrderToSupplierMapper.mapOrder;
 
 @Service
 public class OrderToSupplierService {
 
     private final OrderToSupplierRepo repo;
     private final ProductService productService;
+    private final OrderItemService orderItemService;
+    private final SupplierService supplierService;
 
-    public OrderToSupplierService(OrderToSupplierRepo repo, ProductService productService) {
+    public OrderToSupplierService(OrderToSupplierRepo repo, ProductService productService, OrderItemService orderItemService, SupplierService supplierService) {
```

Comment:
Here you can delete the constructor and annotate the class with `@RequiredArgsConstructor`.